live-examples/js-examples/object/object-entries.html

```diff
@@ -8,7 +8,12 @@
 // expected output: Array ["2", "c"]
 
 const object3 = { 100: 'a', 2: 'b', 7: 'c' };
-console.log(Object.entries(object3)[0]);
+//Object.entries() may or may not  return the objects in a particular order, Sort is needed
```

Comment:
Suggested rephrase of the above

```suggestion
/* Object.entries() does not guarantee a particular order. If order is important,
   use in conjunction with the .sort function */
```